suite-native/receive/src/components/ReceiveAccount.tsx

```diff
@@ -65,10 +65,11 @@ export const ReceiveAccount = ({ accountKey, tokenContract }: AccountReceiveProp
                     isReceiveApproved={isReceiveApproved}
                     isUnverifiedAddressRevealed={isUnverifiedAddressRevealed}
                     onShowAddress={handleShowAddressAndRemoveButtonRequests}
+                    isConfirmOnTrezorReady={isConfirmOnTrezorReady}
                 />
             </VStack>
 
-            {isConfirmOnTrezorReady && <ConfirmOnTrezorImage />}
+            {isConfirmOnTrezorReady && isUnverifiedAddressRevealed && <ConfirmOnTrezorImage />}
```

Comment:
This is already part of `isConfirmOnTrezorReady`

Reply:
changed in b440128